src/query/fuzzy_query.rs

```diff
@@ -0,0 +1,144 @@
+use common::BitSet;
+use core::SegmentReader;
+use fst::Automaton;
+use levenshtein_automata::{LevenshteinAutomatonBuilder, DFA};
+use query::BitSetDocSet;
+use query::ConstScorer;
+use query::{Query, Scorer, Weight};
+use schema::{Field, IndexRecordOption, Term};
+use termdict::{TermDictionary, TermStreamer};
+use Result;
+use Searcher;
+
+/// A Fuzzy Query matches all of the documents
+/// containing a specific term that is with in
+/// Levenshtein distance
+#[derive(Debug, Clone)]
+pub struct FuzzyQuery {
```

Comment:
Please renamed it to `FuzzyTermQuery` to make it clear that it works over a single term. This is to avoid confusion with how we will try and handle queries with multiple terms.